docs/maintenance/recompress-storage.md

```diff
@@ -0,0 +1,202 @@
+---
+id: recompress-storage
+title: Recompressing archival node storage
+sidebar_label: Recompressing archival node storage
+sidebar_position: 4
+description: How to recompress archival node’s storage to reduce it’s size.
+---
+
+# Recompressing archival node storage
+
+The neard 1.26 release introduced various improvements to node’s
+storage layer.  Among better performance, the changes result in
+*significant* reduction of the storage on archival nodes: from 7.9 TB
+to 3.1 TB on mainnet and from 4.3 TB to 1.5 TB on testnet.
+
+However, to take full advantage of the saving some manual steps need
+to be taken.  This document describes those steps.  Note that if
+you’re not running an archival node, you don’t need to do anything.
+
+One of the improvements is changing of the compression algorithm used
+by the databases.  Because of that, the whole process is referred to
+as ‘recompressing the storage’.
+
+
+## Performing the operation
+
+The operation can be performed at any time.  That is, it does not need
+to happen directly after the node upgrade.  In fact, it might be
+better to schedule it some number of days after binary upgrade.  Such
```

Comment:
@mina86 When would you ideally like these manual operation to occur on archival nodes during this release cycle?

Reply:
That’s tricky to answer because it depends on one’s priorities.  If we’re ignoring cost of storage, I’d say a few days after the protocol upgrade has finalised on chain so that everything else is settled.  But it really could be done at the same time as binary upgrade.